addons/sourcemod/scripting/surftimer/replay.sp

```diff
@@ -1380,6 +1380,12 @@ public void Stage_SaveRecording(int client, int stage, char[] time)
 
 	for (int i = startFrame; i < endFrame; i++)
 	{
+		if (i == -1)
+		{
+			LogError("Stage record cannot be saved. Client: \"%L\", startFrame: %d (g_iStageStartFrame: %d), endFrame: %d (g_iRecordedTicks: %d), i: %d, Path/File: %s", client, startFrame, g_iStageStartFrame[client], endFrame, g_iRecordedTicks[client], i, sPath2);
+			break;
```

Comment:
It is useless to skip -1 index as startframe tick, because if startframe is -1 it means that replay was not recorded at all. So you will not get .rec file even if you continue the process i guess